### DISCUSSION TITLE
The text editing options are broken on iPad

### DISCUSSION THREAD

**Discussion by theblixguy** [community/community#125469] [Mobile] [answered]
Here's what it looks like - you basically cannot use any of the edition options as it just closes the screen. It's been broken like this for months.

It's the same problem when I open the "Edit" screen to edit a PR description or the "Add comment" screen to add a PR description. Basically anywhere the app shows these editing options it's broken like this.
![image](https://github.com/community/community/assets/1307667/19620f78-b6b4-4ce2-bd37-1f675ffaf952)
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.163.0 (131249690)
Device: iPad16,6 (iOS 17.5.1)
</details>

**Comment by taki-on** [comment, marked as answer]
@theblixguy I guess you are using external keyboards. The issue is known to us, and we are currently experimenting with a solution optimized for iPad. Please stay tuned!

**Reply by theblixguy** [reply]
Yep it's with my Magic Keyboard connected, although I also have the same issue when using Universal Control. 

Thanks for the update, hope this gets fixed soon so I can continue using this app as part of my workflow.
